sale_order_line_menu/views/sale_order_line_views.xml

```diff
@@ -25,6 +25,7 @@
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">sale.order.line</field>
         <field name="view_mode">tree,pivot</field>
+        <field name="domain">[("display_type","=",False)]</field>
```

Comment:
Is this really wanted ? Shouldn't it be with a filter instead ?

Reply:
This change differs from v14

Reply:
OK, I think functionality could be replaced by a filtered applied by default. I put this because displaying in this view section or note lines with a lot of quantity and amount information is a little bit weird. @dreispt what do you think?

And it's correct, this change differs from v14, as I've mentioned when opening this PR:

> 
_Also added, in a separate commit, a bugfix that excludes order lines that belongs to section or note lines. This commit could be excluded from here, and moved to a standalone PR, if needed_

Since this change is in another commit, it could be cherry-picked later, is it right?

Reply:
I see @rousseldenis point. There might be people who expect of need to see those special lines.
I suggest you make this a search filter, that is enabled by default.
This way you get the default behavior most people need (IMO) without imposing it.

Reply:
Comments attended, [FIX] moved to [IMP] as required

Reply:
@rousseldenis @dreispt comments are attended, could you reopen this and merge, if everything's fine? Thanks!